source/agora/utils/PrettyPrinter.d

```diff
@@ -45,7 +45,7 @@ unittest
 
 /// Returns:
 /// A formatting struct for a type, or the value if no such struct exists
-public auto prettify (T) (const ref T input) nothrow
+public auto prettify (T) (auto const ref T input) nothrow
```

Comment:
This is a sneaky change and it should be moved to a separate commit, or removed entirely.